projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/create.py

```diff
@@ -30,7 +30,13 @@ def __init__(self, rest_api_url: str):
 
     @ApiClientErrorDecorator()
     def create_job(
-        self, name: str, team: str, path: str, cloud: bool, local: bool
+        self,
+        name: str,
+        team: str,
+        path: str,
+        cloud: bool,
+        local: bool,
+        sample_job=None,
```

Comment:
don't forget the type hints.

Reply:
not completely sure what type hint to include as we pass a module

Reply:
Google (or chatGPT now) has all the answers. 

For example https://stackoverflow.com/questions/53780735/what-is-the-type-hint-for-a-any-python-module :) 

Though if we use path it might be moot point.